cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/camera/Camera1Adapter.kt

```diff
@@ -1,4 +1,5 @@
 @file:Suppress("deprecation")
+/* camera1 is deprecated, but still our best option for android 5.0 */
```

Comment:
Can you comment on why?

Reply:
Camera2 is broken in android API 21. The YUV image it returns is in an incorrect format, which affects a very limited number of older devices (visible in manual testing on a Samsung Galaxy Note 3).

CameraX (which uses camera2 under the hood) still has alpha dependencies, which merchants have expressed hesitancy about integrating.

I'll add this to the comment.